Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -386,7 +386,6 @@ class TabTrayViewController: UIViewController {
 
 extension TabTrayViewController: Themeable {
      @objc func applyTheme() {
-         overrideUserInterfaceStyle =  ThemeManager.instance.userInterfaceStyle
```

Comment:
There are some Ecosia comments some lines down, shall we do the same for this file and comment instead of removing?

Reply:
as it's a change from FF that I just applied, a comment won't be needed

Reply:
:+1: